### PR TITLE
[stable/kubernetes-dashboard] Add --enable-skip-login option/parameter to chart

### PR DESCRIPTION
**Pull request by Brightside56**
#### What this PR does / why we need it:
PR adds support of new --enable-skip-login option/flag to chart, which is supported in latest release of dashboard

https://github.com/kubernetes/dashboard/releases/tag/v1.10.1 

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by desaintmartin** [comment]
Thanks! Could you sign your commit ?
/ok-to-test

File: stable/kubernetes-dashboard/templates/deployment.yaml [hunk 1]
```diff
@@ -39,6 +39,9 @@ spec:
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         args:
+{{- if .Values.enableSkipLogin }}
+          - --enable-skip-login
+{{- else }}
```
Comment (desaintmartin) [inline stable/kubernetes-dashboard/templates/deployment.yaml]: end  instead of else?

**Review by desaintmartin — CHANGES_REQUESTED** [review]

File: stable/kubernetes-dashboard/templates/deployment.yaml (hunk 1 above)
Comment (Brightside56) [reply stable/kubernetes-dashboard/templates/deployment.yaml]: Yes

**Comment by Brightside56** [comment]
@desaintmartin , now should be fine, sorry, end of working day.

**Comment by desaintmartin** [comment]
/lgtm
Thanks for the contribution :)

**Pull request merged**